frontend/packages/console-app/src/components/modals/resource-limits/ResourceLimitsModalLauncher.tsx

```diff
@@ -0,0 +1,83 @@
+import * as yup from 'yup';
+import { Formik } from 'formik';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { K8sKind, k8sPatch, K8sResourceKind } from '@console/internal/module/k8s';
+import { limitsValidationSchema } from '@console/dev-console/src/components/import/validation-schema';
+import { getLimitsDataFromResource, getResourceLimitsData } from '@console/shared/src';
+import { createModalLauncher, ModalComponentProps } from '@console/internal/components/factory';
+
+import ResourceLimitsModal from './ResourceLimitsModal';
+
+export type ResourceLimitsModalLauncherProps = {
+  model: K8sKind;
+  resource: K8sResourceKind;
+  close?: () => void;
+} & ModalComponentProps;
+
+const ResourceLimitsModalLauncher: React.FC<ResourceLimitsModalLauncherProps> = (props) => {
+  const { t } = useTranslation();
+
+  const handleSubmit = (values, actions) => {
+    const {
+      limits: { cpu, memory },
+    } = values;
+    const resources = getResourceLimitsData({ cpu, memory });
+    k8sPatch(props.model, props.resource, [
+      {
+        op: 'replace',
+        path: `/spec/template/spec/containers/0/resources`,
```

Comment:
@bmignano since here we will be editing the resourceLimits of the first container only so I think it makes sense to show the name of the container so that the user is aware about that. Wdyt?

Reply:
@divyanshiGupta agree, it makes sense to show the container name!

Reply:
@bmignano  ![Screenshot from 2021-03-29 16-28-28](https://user-images.githubusercontent.com/22490998/112827543-1c63a880-90ac-11eb-93c5-3c85322bca95.png)
 please let me know if this looks good

Reply:
LGTM! @debsmita1